src/test/README.md

```diff
@@ -19,7 +19,8 @@ and tests weren't explicitly disabled.
 Assuming the build directory is named `build`, the unit tests can be run
 with `ctest --test-dir build`, which includes unit tests from subtrees.
 
-To run the unit tests manually, launch `build/src/test/test_bitcoin`. To recompile
+To run the unit tests manually, launch `build/src/test/test_bitcoin`
+(assuming `build` is your build directory). To recompile
```

Comment:
You're adding `(assuming build is your build directory)` here, but 3 lines up it already says `Assuming the build directory is named build,`.